test/test_store.py

```diff
@@ -191,17 +189,16 @@ def test_get_data_opener_ids_with_valid_data_id(self):
 
 
 class CmemsDataStoreParamsTest(unittest.TestCase):
-
     def test_store_for_cmems_credentials(self):
         params = {"cmems_username": "", "cmems_password": ""}
         with self.assertRaises(Exception) as e:
             CmemsDataStore(**params)
         self.assertEqual(
-            "CmemsDataStore needs cmems credentials to "
-            "be provided either as "
-            "environment variables CMEMS_USERNAME and "
-            "CMEMS_PASSWORD, or to be "
-            "provided as store params cmems_username and "
-            "cmems_password",
+             "CmemsDataStore needs cmems credentials to "
+                "be provided either as "
+                "environment variables COPERNICUSMARINE_SERVICE_USERNAME and "
+                "COPERNICUSMARINE_SERVICE_PASSWORD, or to be "
```

Comment:
Why did you change these names? I actually liked the old ones better.

Reply:
Created a new issue for this, https://github.com/xcube-dev/xcube-cmems/issues/51. Will be handled in the next PR